docs/en/php/developer.md

```diff
@@ -56,18 +56,20 @@ Or if you have only `debug` option in the `pool` section you can use short synta
 http:
   pool.debug: true
 ```
-
 > **Note**
 > Every plugin in RoadRunner that creates workers has a `pool` section in which you can activate debug mode.
 
-In this mode, RoadRunner does not create a worker when it starts up, regardless of the configuration settings. Instead,
-it waits for requests to arrive and creates workers accordingly. After the response, RoadRunner stops and removes the
-worker, allowing you to make changes to your codebase and reload it automatically.
-
-If you send 2-3-n parallel requests, RoadRunner will create 2-3-n workers to handle those requests simultaneously.
-Similarly, when using the `Jobs` plugin and the `Jobs` consumers, each consumed message will create a worker.
 
-More info: [debug](debugging.md)
+> **Warning**
+> When using the `pool.debug` option in RoadRunner, it is important to note that the `pool.num_workers` section will not
```

Comment:
I guess we might note here, that all pool section would work differently. All options will be ignored (supervisor, max_jobs, etc)